packages/relay/conf/config.json

```diff
@@ -2,6 +2,9 @@
   "participants": [
     "001", "002", "003", "004", "005", "006", "007", "008"
   ],
+  "adminsNetworks": [
+    "10.0.0.5/24", "127.0.0.1/24"
```

Comment:
Not sure if that's the best default config. I think it should be either off by default: 0.0.0.0/0, or empty to ensure users will specify the admins network